src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -101,30 +98,37 @@ export const RegistrationFormActions = ({ tabNumber, setTabNumber, validateForm
           </Box>
         )}
 
-        <Box sx={{ gridArea: 'support-button', display: 'flex', justifyContent: 'start' }}>
+        <Box sx={{ gridArea: 'support-button', display: 'flex', justifyContent: 'space-evenly' }}>
           <Button
+            sx={{
+              borderRadius: '15px 0px 15px 0px',
+              bgcolor: 'white',
+              border: '2px solid #4A7AC1',
+              borderRight: '5px solid #4A7AC1',
+              borderLeft: '5px solid #4A7AC1',
+            }}
+            size="small"
             data-testid={dataTestId.registrationWizard.formActions.openSupportButton}
             onClick={toggleSupportModal}>
-            {t('common.support')}
+            {t('registration.curator_support')}
           </Button>
         </Box>
         {!isLastTab ? (
-          <>
-            <Box
-              sx={{
-                gridArea: 'save-button',
-                display: 'flex',
-                justifyContent: 'end',
-              }}>
-              <LoadingButton
-                variant="outlined"
-                loading={isSaving}
-                data-testid={dataTestId.registrationWizard.formActions.saveRegistrationButton}
-                onClick={async () => await saveRegistration(values)}>
-                {t('common.save')}
-              </LoadingButton>
-            </Box>
-            <Box sx={{ gridArea: 'next-button', display: 'flex', justifyContent: 'end' }}>
+          <Box
+            sx={{
+              gridArea: 'save-button',
+              display: 'flex',
+              justifyContent: 'end',
+              alignItems: 'center',
+            }}>
+            <LoadingButton
+              variant="outlined"
+              loading={isSaving}
+              data-testid={dataTestId.registrationWizard.formActions.saveRegistrationButton}
+              onClick={async () => await saveRegistration(values)}>
+              {t('common.save')}
+            </LoadingButton>
+            <Box sx={{ gridArea: 'next-button' }}>
               <Tooltip title={t('common.next')}>
```

Comment:
ser ut som vi kan spare oss en tag og ett innrykk her?
```suggestion
              <Tooltip title={t('common.next')} sx={{ gridArea: 'next-button' }}>
```